figures/metrics.py

```diff
@@ -273,12 +275,17 @@ def get_site_mau_current_month(site):
     generation. When we update for MAU 2G, we will be able to make the query
     more efficient by pulling unique users from a single table used for live
     capture.
+
+    IMPORTANT: This query gets slow as the data grow. We've refactored the code
+    to do this query in the pipeline, storing values to `SiteDailyMetrics.mau`
+
+    TODO: Move this method to figures.mau
     """
     month_for = datetime.datetime.utcnow()
     site_sm = figures.sites.get_student_modules_for_site(site)
     curr_sm = site_sm.filter(modified__year=month_for.year,
                              modified__month=month_for.month)
-    return curr_sm.values('student__id').distinct().count()
+    return curr_sm.values('student__id').distinct()
```

Comment:
We do this so we're able to use this function to retrieve the distinct ids, then we can get the count from the querset for when we need the count